librz/core/cmd_descs/cmd_debug.yaml

```
@@ -373,6 +373,67 @@ commands:
             type: RZ_CMD_ARG_TYPE_NUM
           - name: len
             type: RZ_CMD_ARG_TYPE_NUM
+  - name: de
+    summary: Manage ESIL watchpoints
+    subcommands:
+      - name: de
+        summary: Add ESIL watchpoint
+        cname: cmd_debug_esil_add
+        args:
+          - name: perm
+            type: RZ_CMD_ARG_TYPE_STRING
+          - name: kind
+            type: RZ_CMD_ARG_TYPE_CHOICES
+            choices:
+              - reg
+              - mem
+          - name: expression
+            type: RZ_CMD_ARG_TYPE_STRING
+        details:
+          - name: Examples
+            entries:
+              - text: de
+                arg_str: "r reg rip"
+                comment: Stop when reads RIP register
+              - text: de
+                arg_str: "rw mem ADDR"
+                comment: Stop when read or write in ADDR
+              - text: de
+                arg_str: "w r rdx"
+                comment: Stop when rdx register is modified
+              - text: de
+                arg_str: "x m FROM..TO"
+                comment: Stop when rip in range
+      - name: de-
```

```suggestion
      - name: de-*
```

I think this is better for consistency. Usually `<cmd>-` is for deleting one item, `<cmd>-*` for all.